install/kubernetes/cilium/Chart.yaml

```diff
@@ -1,9 +1,53 @@
 apiVersion: v2
 name: cilium
+displayName: Cilium
```

Comment:
do you have a link to Chart.yaml schema?. I taked a look here https://helm.sh/docs/topics/charts/, but `displayName` and `links` are not mentioned.

Reply:
I'm not sure if those ones are part of some default main schema but I found them while looking through the artifact hub documentation. Hopefully most tooling is willing to overlook fields they don't understand.

Reply:
oh right :100:, all good :)